server/app/views/templates/base/head.scala.html

```diff
@@ -0,0 +1,33 @@
+@import play.api.Play.current
+@(title: String, description: String )(implicit request: RequestHeader)
+
+
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")" />
+
+<title>@title</title>
+<meta name="description" content="@description">
+<meta name="author" content="47 Degrees">
+<meta property="og:image" content="@routes.Assets.at("images/poster.png")"/>
```

Comment:
Would this result in a relative URL? If so I think we need absolute URLs in all instances.